Drink.java

```diff
@@ -0,0 +1,48 @@
+package com.company;
+
+// It is task1
+//Создаем новые напитки через билдер
+
+public class Drink{
+    private String name;
+    private String description;
+    private double price;
+
+    public String getName(){
+        return name;
+    }
+
+    public String getDescription(){
+        return description;
+    }
+
+    public double getPrice(){
+        return price;
+    }
+
+    private Drink(Builder builder){

```

Comment:
better create Drink in builder not in Drink. Drink should not know about builder